packages/ui-components/src/utils/objectToClassnames.ts

```diff
@@ -1,3 +1,27 @@
+
+import flow from "lodash/fp/flow";
+import filter from "lodash/fp/filter";
+import split from "lodash/fp/split";
+import join from "lodash/fp/join";
+import map from "lodash/fp/map";
+import uniq from "lodash/fp/uniq";
+import isEmpty from "lodash/isEmpty";
+import negate from "lodash/negate";
+
+const appendSuffix = (suffix: string) => (string: string) =>
+  `${string}${suffix}`;
+
+const containerSuffix = appendSuffix("__container");
+
+export const generateContainerClassnames = (className = ""): string =>
```

Comment:
I'm hoping that we don't need this utility. But if we do, it should be moved to it's own file of the same name.

Reply:
removed, I think we can avoid it if we just change the classNames we pass in from console, which can be done in the integration. from what i see in console, it's only used in one form (deleteOrgModal)